app/components/Login/Login.jsx

```diff
@@ -113,7 +120,10 @@ export default class Login extends React.Component {
                 })
                 .catch((err) => {
                     console.error(err.stack);
-                    this.setState({ errorMessage: err.response.data })
+                    this.setState({ errorMessage: err.response.data, loginMethodType: this.getVaultAuthMethod() })
```

Comment:
I think this can be combined
```js
this.setState({
    errorMessage: `${window.suppliedAuthToken ? 
                       'Login was attempted using a server supplied token. Please contact your network administrator -- ' :  ''}${err.response.data}`
});